### PR TITLE
Add native library reproducible build

### PR DESCRIPTION
**Pull request by rmoreliovlabs**

### CODE REVIEW

File: native/1.3.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ apt-get install -y -o APT::Install-Suggests="true" autoconf && \
+ curl "https://dl.google.com/go/"$GOLANG -o $GOLANG -# && \
+ echo "512103d7ad296467814a6e3f635631bd35574cab3369a97a323c9a585ccaa569  go1.13.5.linux-amd64.tar.gz" > goChecksum.txt && \
+ cat goChecksum.txt && \
+ sha256sum -c goChecksum.txt && \
+ tar -xvf $GOLANG && mkdir -p $GOCUSTOM && mv go $GOCUSTOM
+
+RUN apt-get install -y gcc-aarch64-linux-gnu
+
+RUN git clone https://github.com/rsksmart/native.git /code/native
```
Comment (Vovchyk) [inline native/1.3.0/Dockerfile:28]: `git clone` fetches all the stuff. I'd suggest using something like [this](https://github.com/rsksmart/reproducible-builds/blob/master/rskj/4.1.0-hop/Dockerfile#L8-L12)

File: native/1.3.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+WORKDIR /code/native
+RUN git checkout tags/1.3.0 -b 1.3.0
+
+#Cloning native's repo
+#RUN git clone https://github.com/rsksmart/native.git
```
Comment (Vovchyk) [inline native/1.3.0/Dockerfile:34]:
```suggestion

```
not needed?

**Review by Vovchyk — CHANGES_REQUESTED** [review]

File: native/1.3.0/Dockerfile (hunk 1 above)
Comment (rmoreliovlabs) [reply native/1.3.0/Dockerfile]: Done

File: native/1.3.0/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Native Library 1.3,0
+
+* Source: https://github.com/rsksmart/native
+* Tag: `1.3.0`
+
+## Build
+
+```
+$ docker build --build-arg NATIVE_LIB_VERSION=1.3.0 -t native-1.3.0 .
```
Comment (Vovchyk) [inline native/1.3.0/README.md:9]:
```suggestion
$ docker build -t native-1.3.0 .
```
seems there's no need to pass the arg. Its value is already defined in Dockerfile

File: native/1.3.0/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+## Verify
+
+```
+$ sudo docker run --rm native-1.3.0 sha256sum build/libs/native-1.3.0.jar
```
Comment (Vovchyk) [inline native/1.3.0/README.md:15]:
```suggestion
$ docker run --rm native-1.3.0 sha256sum build/libs/native-1.3.0.jar
```

File: native/1.3.0/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+RUN mkdir /native
+
+RUN git clone --single-branch --depth 1 --branch $NATIVE_LIB_VERSION https://github.com/rsksmart/native.git /native
+
+WORKDIR /native
+
+# Copying jni headers to java /include
+RUN cp -r jniheaders/include $JAVA_HOME
+
+# Expose
```
Comment (Vovchyk) [inline native/1.3.0/Dockerfile:40]:
```suggestion
# Build
```

File: native/1.3.0/README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+cf03d2230ae7cf5349b44ffb3f089193aff17d8b8f6071ff562605d1be99228c  build/libs/native-1.3.0.jar
+```
+
+## (Optional) Extract JAR from image
+
+```
+$ docker run --name temp-container native-1.3.0 /bin/true
+$ docker cp temp-container:build/libs/native-1.3.0.jar ./native-1.3.0.jar
```
Comment (Vovchyk) [inline native/1.3.0/README.md:23]: did this command work for you?

**Review by Vovchyk — CHANGES_REQUESTED** [review]

File: native/1.3.0/README.md (hunk 6 above)
Comment (rmoreliovlabs) [reply native/1.3.0/README.md:23]: Path was wrong just fixed it

**Pull request merged**